index.html

```diff
@@ -0,0 +1,1406 @@
+            <a href="#" class="bnl-a no-underline">events<i class="fas fa-sort-down arrow-down"></i></a></div>
+          <div class="bnl capitalize">
+            <a href="#" class="bnl-a no-underline">business<i class="fas fa-sort-down arrow-down"></i></a>
+          </div>
+          <div class="bnl capitalize">
+            <a href="#" class="bnl-a no-underline">AMAs</a>
+          </div>
+          <div class="bnl capitalize">
+            <a href="#" class="bnl-a no-underline">spaces</a>
+          </div>
+          <div class="bnl capitalize">
+            <a href="#" class="bnl-a no-underline">terms & conditions</a>
+          </div>
+        </li>
+        <li class="black__navbar-links">
+          <div class="bnl capitalize">
+            <a href="#" class="bnl-a  no-underline">about</a>
+          </div>
+          <div class="bnl capitalize">
+            <a href="#" class="bnl-a no-underline">advertise</a>
+          </div>
+          <div class="bnl capitalize">
+            <a href="#" class="bnl-a no-underline">jobs</a>
+          </div>
+          <div class="bnl capitalize">
+            <a href="#" class="bnl-a no-underline">contact</a>
+          </div>
+        </li>
+      </ul>
+      <ul class="white__navbar">
+        <li class="white__navbar-links">
+          <div class="wnl uppercase">
+            <a href="#" class="wnl-a no-underline">
+              latest
+            </a>
+          </div>
+          <div class="wnl uppercase">
+            <a href="#" class="wnl-a no-underline">
+              hard fork
+            </a>
+          </div>
+          <div class="wnl uppercase">
+            <a href="#" class="wnl-a no-underline">
+              plugged
+            </a>
+          </div>
+          <div class="wnl uppercase">
+            <a href="#" class="wnl-a no-underline">
+              readme
+            </a>
+          </div>
+          <div class="wnl uppercase">
+            <a href="#" class="wnl-a no-underline">
+              growth quarters
+            </a>
+          </div>
+          <div class="wnl uppercase neural">
+            <a href="#" class="wnl-a no-underline">
+              neural
+            </a>
+          </div>
+
+        </li>
+
+        <li class="white__navbar-container__search-and-menu">
+          <div class="white__navbar-container__search">
+            <i class="fas fa-search"></i>
+          </div>
+          <div class="white__navbar-container__menu">
+            <i class="fas fa-bars"></i>
+          </div>
+        </li>
+      </ul>
+    </section>
+  </nav>
+  <main>
+    <section class="wrapper">
+      <div class="trigrid">
+        <div class="trigrid-article1">
+          <img src="assets/tnw.svg" class="tglogo" alt="">
+          <div class="tga-date">
+            <p class="tga-date-p">
+            </p>
+          </div>
+          <div class="tga-info">
+            <a href="#" class="tga-category bold700 uppercase no-underline">
+              insights
```

Comment:
- [ ] This section is being covered by the navbar. Kindly address that. The TNW logo also would not be displayed in the medium screen size.

![image](https://user-images.githubusercontent.com/61479648/88212464-b164f680-cc2d-11ea-817f-a99dfc835303.png)